src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Create.cs

```diff
@@ -428,9 +428,19 @@ public async Task UpdateAddFile()
         public void Windows_ZipWithInvalidFileNames_ThrowsException(string zipName, string paramName)
```

Comment:
The name of this test method is now wrong. But can we just delete these tests now?

Reply:
We don't need to delete it, I will fix the name tho.

Reply:
It's just duplicating your new tests right?